include/glow/Graph/Node.h

```diff
@@ -257,6 +261,13 @@ class Node : public Named,
   /// \returns the n'th result of the node.
   const NodeValue getNthResult(unsigned idx) const;
 
+  /// \returns the function holding this node.
+  /// If that node does not belong to any function, this
+  /// is nullptr.
+  Function *getParent() const { return parent_; }
```

Comment:
since it's a const method, I think it's a bit confusing to return internals of the Node in here. (is it possible to make that const Function *?)

Reply:
We could. Technically, changing the function does not change the node and returning a pointer does not change the value of parent_ either.
Anyhow, I agree it can be confusing.

Reply:
(Bottom line I am going to use const :))

Reply:
@rdzhabarov, looking at the IR Instruction, the getParent method has exactly the same signature:
  /// \returns parent of current instruction.
  IRFunction *getParent() const { return F_; }

I can do the clean-up in both places in a follow-up PR.

What do you think?

Reply:
I'd fix here, just on a separate commit, but up to you. Good catch!